src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -10302,6 +10302,24 @@ instruct array_equalsC(iRegP_R11 ary1, iRegP_R12 ary2, iRegI_R10 result,
   ins_pipe(pipe_class_memory);
 %}
 
+// fast ArraysSupport.vectorizedHashCode
+instruct arrays_hashcode(iRegP_R11 ary, iRegI_R12 cnt, iRegI_R10 result, immI basic_type,
+                         iRegLNoSp tmp1, iRegINoSp tmp2,
+                         iRegINoSp tmp3, iRegLNoSp tmp4, rFlagsReg cr)
```

Comment:
Maybe declare `tmp2` and `tmp3` as `iRegLNoSp`? I see these two are used as 64-bit registers in `C2_MacroAssembler::arrays_hashcode`.

Reply:
Agreed.

Reply:
Fixed.